scripts/clr-icons-svg.js

```diff
@@ -7,19 +7,10 @@
 const writeSVGIcons = require('./write-svg-icons');
 const shell = require('shelljs');
 
-const SHAPE_SETS = [
-  'core-shapes',
-  'commerce-shapes',
-  'essential-shapes',
-  'media-shapes',
-  'social-shapes',
-  'technology-shapes',
-  'travel-shapes',
-  'chart-shapes',
-];
+const SHAPE_SETS = ['core', 'commerce', 'essential', 'media', 'social', 'technology', 'travel', 'chart'];
```

Comment:
Could you add `text-edit-shapes` here?

Reply:
Thanks for catching that. Should be fixed now.